src/components/Nav/Login/Login.js

```diff
@@ -0,0 +1,204 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import theme from '../../../styles/theme';
+import { flexAlignCenter } from '../../../styles/mixin';
+
+const Login = props => {
+  const { modalState, closeModal } = props;
+  const [userData, setUserData] = useState({
+    id: '',
+    pw: '',
+  });
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setUserData(prev => ({ ...prev, [name]: value }));
+  };
+
+  const disabled = userData.id.length * userData.pw.length > 1 ? false : true;
+
+  const REST_API_KEY = 'da3ecebce76c275513b1f3b4d37432dc';
+  const REDIRECT_URI = 'http://localhost:3000/oauth';
+  const KAKAO_OAUTH_URL = `https://kauth.kakao.com/oauth/authorize?response_type=code&client_id=${REST_API_KEY}&redirect_uri=${REDIRECT_URI}&response_type=code`;
+
+  if (!modalState) return null;
+  return (
```

Comment:
```suggestion
  if (!modalState) return null;

  return (
```
다른 문맥들끼리는 공백을 통해서 가독성에 신경써주세요